src/test/java/org/cactoos/list/ListEnvelopeTest.java

```diff
@@ -159,9 +159,9 @@ public void returnsSubListWithUnsupportedAdd() {
     }
 
     @Test
-    public void getsPreviousIndex() {
+    public void mustReturnPreviousIndex() {
         new Assertion<>(
-            "List iterator returns incorrect previous index",
+            "List Iterator must return previous index",
```

Comment:
@fanifieiev why is `ListIteratorOf` tested in `ListEnvelopeTest`? I think those tests should be in their own file (and removed from here).

Reply:
@victornoel I agree with your concern, I just left them where they were previously.

Reply:
@fanifieiev so please move them in their own file to properly identify what is being tested